core/src/test/java/com/linecorp/armeria/client/DnsMetricsTest.java

```diff
@@ -147,7 +154,7 @@ void timeout() {
                         () -> client2.execute(RequestHeaders.of(HttpMethod.GET, "http://foo.com"))
                                      .aggregate().join());
                 assertThat(cause.getCause()).isInstanceOf(UnprocessedRequestException.class);
-                assertThat(cause.getCause().getCause())
+                assertThat(Throwables.getRootCause(cause))
```

Comment:
👍

Reply:
The root cause is `UnknownHostException`?
https://ci.appveyor.com/project/line/armeria/builds/36552679/job/omd5r3ft1vwdxa44#L595

Reply:
No, it's not? 😄